E2-Breakdown-Money/src/main/java/com/redelles/e2/Numbers.java

```diff
@@ -0,0 +1,76 @@
+package com.redelles.e2;
+
+import java.util.Scanner;
+
+public class Numbers {
+
+    public static Integer scanNumber(){
+
+        Integer numberInput = -1;
+
+        do {
+            try {
+
+                Scanner scan = new Scanner(System.in);
+                numberInput = scan.nextInt();
+
+            } catch (Exception e) {
+                System.out.println("REMEMBER This application only works with numbers.");
+            }
+        }while( numberInput == -1);
+
+        return numberInput;
+    }
+
+    public static Integer getAmount(){
```

Comment:
getAmount and getQty are the same behaviour only changes the message to show, it can be extracted as method